public/app/features/alerting/unified/components/receivers/grafanaAppReceivers/onCall/useOnCallIntegration.tsx

```diff
@@ -100,10 +101,18 @@ export function useOnCallIntegration() {
           return true;
         } catch (error) {
           if (isFetchError(error) && error.status === 409) {
-            return 'Integration of this name already exists in OnCall';
+            return t(
+              'alerting.irm-integration.integration-name-exists',
+              'Integration of this name already exists in IRM'
+            );
           }
 
-          notifyApp.error('Failed to validate OnCall integration name. Is the OnCall API available?');
+          notifyApp.error(
+            t(
+              'alerting.irm-integration.validation-failed',
+              'Failed to validate IRM integration name. Is the OnCall API available?'
+            )
+          );
```

Comment:
Just checking - the second part is still supposed to say `OnCall`, yes?

Reply:
Yes, we have a unified IRM plugin but this single plugin uses OnCall and Incident APIs